packages/react/ds/src/icon/icon.tsx

```diff
@@ -24,6 +27,7 @@ export function Icon({
   ariaHidden,
   ariaLabel,
   inline = false,
+  className = '',
```

Comment:
I wonder if we need this. We probably don't need it if it's only required to change the colour.

Reply:
we can get around it in this instance, maybe for other components we might need to change the colour but I can guess we can make the changes when/if we come across the problem